app/controllers/trainees/funding/training_initiatives_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Trainees
+  module Funding
+    class TrainingInitiativesController < ApplicationController
+      before_action :authorize_trainee
+
+      def edit
+        @training_initiatives_form = ::Funding::TrainingInitiativesForm.new(trainee)
+      end
+
+      def update
+        @training_initiatives_form = ::Funding::TrainingInitiativesForm.new(trainee, params: trainee_params, user: current_user)
+
+        save_strategy = trainee.draft? ? :save! : :stash
+
+        if @training_initiatives_form.public_send(save_strategy)
+          redirect_to review_draft_trainee_path(trainee)
```

Comment:
To be updated once the bursary form is in. Captured in this ticket: https://trello.com/c/NI1M2o1g/1992-m-funding-ui-joining-everything-together